docs/cookbook/pii_tutorial.md

```diff
@@ -0,0 +1,260 @@
+
+#### Step 2: Configuring Eunomia
+
+Next, configure Eunomia to identify and handle the following types of PII within the tickets:
+
+- Email addresses
+- Names and surnames
+- Phone numbers
+
+```python
+eunomia = Orchestra(
+    instruments=[
+        PiiInstrument(entities=["EMAIL_ADDRESS", "PERSON", "PHONE_NUMBER"], edit_mode="replace"),
+    ]
+)
+```
+
+#### Step 3: Implementing the Final Solution
+
+Finally, use the following code to clean PII information from the tickets:
+=== "Code"
+    ```python
+    tickets = [
+        """Ticket ID: #IT-2024-0001
+        User Information:
+        Name: John Doe
+        Email: john.doe@example.com
+        Phone Number: +1 (555) 123-4567
+        Issue Description: The user cannot connect to the VPN.""",
+        
+        """Ticket ID: #IT-2024-0002
+        User Information:
+        Name: Jane Smith
+        Email: jane.smith@example.com
+        Phone Number: +1 (555) 987-6543
+        Issue Description: The application crashes on startup."""
+    ]
+
+    tickets_cleaned = []
+
+    for ticket_text in tickets:
+        # Clean the PII from the current ticket
+        ticket_text_cleaned = eunomia.run(ticket_text)
+
+        # Append the sanitized ticket to the cleaned list
+        tickets_cleaned.append(ticket_text_cleaned)
+
+    for i, cleaned_ticket in enumerate(tickets_cleaned, start=1):
+        print(f"Cleaned Ticket {i}:\n{cleaned_ticket}\n")
+    ```
+=== "Output"
+    ```bash
+    Cleaned Ticket 1:
+    Ticket ID: #IT-2024-0001
+        User Information:
+        Name: <PERSON>: <EMAIL_ADDRESS>
+        Phone Number: +1 <PHONE_NUMBER>
+        Issue Description: The user cannot connect to the VPN.
+
+    Cleaned Ticket 2:
+    Ticket ID: #IT-2024-0002
+        User Information:
+        Name: <PERSON>
+        Email: <EMAIL_ADDRESS>
+        Phone Number: +1 <PHONE_NUMBER>
+        Issue Description: The application crashes on startup.
+    ```
+
+---
+
+## Secure Retrieval of User Information in Chatbots
+
+### ID-Based Access Control: Customizing Data Accessibility
+
+Eunomia’s ID-Based Access Control (IDBAC) provides precise control over which information is accessible based on the identity of the requestor.
+
+#### Key Components of IDBAC
+
+1. **ID-Based Mechanism**: Each user or entity is assigned a unique identifier.
+2. **Document Ownership**: Every document is linked to the unique ID of its owner.
+3. **Global PII Masking**: Specific types of PII are consistently masked, regardless of the viewer.
+4. **Owner-Specific PII Access**: Owners can view their own PII while it remains masked for others.
+
+#### Step 1: Setup
+
+```python
+from eunomia.orchestra import Orchestra
+from eunomia.instruments import PiiInstrument, IdbacInstrument
+```
+
+You can find out more about the components used at:
+
+- [Orchestra docs](../api/orchestra.md)
+- [PiiInstrument docs](../api/instruments/pii_instrument.md)
+- [IdbacInstrument docs](../api/instruments/idbac_instrument.md)
```

Comment:
Bullet points take a lot of space here, I would rather list them in the same paragraph